spec/requests/api/v1/posters_request_spec.rb

```diff
@@ -136,8 +136,8 @@
     new_poster = JSON.parse(response.body, symbolize_names: true)
 #require 'pry'; binding.pry
     expect(response).to be_successful
-    expect(new_poster[:name]).to eq(new_name)
-    expect(new_poster[:name]).not_to equal(original_name)
+    expect(new_poster[:data][:attributes][:name]).to eq(new_name)
```

Comment:
Now that we've run the return through the serializer, we need to dig into the `:data` KVP to access the values